feature-configs/base/sctp/is_ready.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+oc wait mcp/worker --for condition=updating --timeout 3s
```

Comment:
@fedepaol there's a mcp.Status.Configuration.Source slice that holds references to all machineConfigs that are rendered in the MCP's aggregated machine config.

maybe check that sctp is in that list AND condition=updated ? then you might not have to add timing related logic here.

Reply:
Done, not sure if the condition check can be done in a more elegant way